mne/source_space.py

```diff
@@ -2306,24 +2327,105 @@ def get_volume_labels_from_aseg(mgz_fname):
 
     Returns
     -------
-    label_names : list of str
+    label_names_sorted : list of str
         The names of segmented volumes included in this mgz file.
-
+    label_colors_sorted : list of str
+        The RGB colors of the labels included in this mgz file.
     Notes
     -----
     .. versionadded:: 0.9.0
     """
     import nibabel as nib
+    import pandas as pd
 
     # Read the mgz file using nibabel
     mgz_data = nib.load(mgz_fname).get_data()
 
     # Get the unique label names
     lut = _get_lut()
+
     label_names = [lut[lut['id'] == ii]['name'][0].decode('utf-8')
                    for ii in np.unique(mgz_data)]
-    label_names = sorted(label_names, key=lambda n: n.lower())
-    return label_names
+    label_colors = [[lut[lut['id'] == ii]['R'][0],
+                     lut[lut['id'] == ii]['G'][0],
+                     lut[lut['id'] == ii]['B'][0],
+                     lut[lut['id'] == ii]['A'][0]]
+                    for ii in np.unique(mgz_data)]
+    label_names_sorted = sorted(label_names, key=lambda n: n.lower())
+
+    p = pd.DataFrame({'label_names': label_names,
+                     'label_colors': label_colors}).sort('label_names')
```

Comment:
you don't need pandas for this.

order = np.argsort(label_names)
label_names = [label_names[k] for k in order]
label_colors = [label_colors[k] for k in order]

and not need to convert back to list